LemonUI/Menus/NativeSubmenuItem.cs

```diff
@@ -29,6 +28,16 @@ public NativeSubmenuItem(NativeMenu menu, NativeMenu parent) : base(menu.Subtitl
             Menu.Parent = parent ?? throw new ArgumentNullException(nameof(parent));
             Activated += NativeSubmenuItem_Activated;
         }
+        /// <summary>
```

Comment:
Isn't this already present as part of the merge of #25?

Reply:
Yeah this is wierd two I've do file diff but it's turn out they are identical.